spec/javascript/orangelight/availability.spec.js

```diff
@@ -392,6 +392,53 @@ describe('AvailabilityUpdater', function() {
     expect(cdl_element.textContent).toContain('Online');
   })
 
+  test('special case for Marquand Stacks items reported as unavailable', () => {
+    document.body.innerHTML = '<table class="availability-table">' +
+      '<tbody>' +
+      '  <tr class="holding-block">' +
+      '    <td class="library-location" data-holding-id="22555592710006421">' +
+      '      <span class="location-text" data-location="true" data-holding-id="22555592710006421">Marquand Library - Marquand Library</span>' +
+      '    </td>' +
+      '    <td class="holding-call-number"></td>' +
+      '    <td class="holding-status" data-availability-record="true" data-record-id="99101378413506421" data-holding-id="22555592710006421" data-aeon="false">'  +
+      '      <span class="availability-icon badge badge-secondary" title=""></span>' +
+      '    </td>' +
+      '    <td class="location-services service-conditional" data-open="true" data-requestable="true" data-aeon="false" data-holding-id="22555592710006421">' +
+      '     <a title="View Options to Request copies from this Location" class="request btn btn-xs btn-primary" data-toggle="tooltip" href="/requests/99101378413506421?mfhd=22555592710006421&amp;source=pulsearch">Request</a>' +
+      '    </td>' +
+      '    <td class="holding-details">' +
+      '      <ul class="item-status" data-record-id="99101378413506421" data-holding-id="22555592710006421"></ul>' +
+      '    </td>' +
+      '  </tr>' +
+      '</tbody>' +
+     '</table>';
+
+    const availability_response = {
+      "99101378413506421":{
+        "22555592710006421":{
+          "on_reserve":"N",
+          "location":"marquand$stacks",
+          "label":"Marquand Library - Marquand Library",
+          "status_label":"Unavailable",
+          "copy_number":null,
+          "cdl":false,
+          "temp_location":false,
+          "id":"22555592710006421"
+        }
+      }
```

Comment:
Can you aslo change the availability response in the spec? 
"99101378413506421": {
"22555592710006421": {
"on_reserve": "N",
"location": "marquand$stacks",
"label": "Marquand Library - Remote Storage: Marquand Use Only",
"status_label": "Unavailable",
"copy_number": null,
"cdl": false,
"temp_location": false,
"id": "22555592710006421"
}
}

Reply:
Good catch. Done.